app/Nova/WaitingList.php

```diff
@@ -51,6 +52,8 @@ public function fields(Request $request)
                 1 => 'ATC Training',
                 2 => 'Pilot Training',
             ])->displayUsingLabels()->rules(['required'])->sortable(),
+
```

Comment:
We need to account for the slug within this form somewhere, or give it a sensible default based on the name